README.md

```diff
@@ -21,20 +28,35 @@ This projects requires node version `7.1.0` or newer.
 Running `nin new <dirname>` will create the specified directory and initialize a new nin project inside.
 Running `nin run` inside the newly created project will make it accessible on http://localhost:8000.
 
-## nin is now node-based!
-
-Nin just became node-based instead of layer-based.
-Here is a list of gotchas to watch out for if you are used to layer-based nin:
+## Shaders
 
-- Changing res/graph.json does *not* trigger updates in a running nin instance.
+Create a shader by clicking `Generate -> Shader Node` in the frontend menu.
+It will be placed in the folder `src/shaders/nameOfTheShader`.
+To get livereload on shader change, you shader must be specified in the options object of your node in `res/graph.json`,
+the shader generator will do this for you.
```

Comment:
Actually it won't, but that's a bug so it is fine that we say so here. I can fix the bug.